canonical-websites/partners.ubuntu.com
# Fix global nav

With a couple of other drive-bys:

- Add a migration to bring the database up-to-date with the state of the models, so we no longer see the "Your models have changes" warning message
- Upgrade to latest ./run
- Add a new script at `scripts/import-modules.sh`, to simplify the `package.json` a bit

QA
--

- `./run`, see that you don't see a "Your models have changes" message
- Go to http://0.0.0.0:8003/admincms/, create a partner, make sure you publish them and give them a partner page as well
- See your partner appears on the homepage
- Visit the partner's partner page
- See that the global nav works again
